Tests/RegexTests/ParseTests.swift

```diff
@@ -174,12 +174,20 @@ extension RegexTests {
     )))
 
     parseTest("[[:alnum:]]", charClass(posixProp_m(.posix(.alnum))))
+    parseTest("[[:ascii:]]", charClass(posixProp_m(.posix(.alnum))))
```

Comment:
```suggestion
    parseTest("[[:ascii:]]", charClass(posixProp_m(.posix(. ascii))))
```

Reply:
Ack, good catch! I actually realized last night that these tests don't correctly detect failures as it's comparing dump output which we haven't completely filled out yet 😬